src/pages/index.tsx

```diff
@@ -48,15 +49,20 @@ export default function HomePage({ schedule }) {
       return schedule;
     }
 
+    const fuse = new Fuse(schedule, {
+      keys: ['nama_lokasi_vaksinasi', 'wilayah', 'kecamatan', 'kelurahan'],
+      threshold: 0.4
+    });
+
+    const results = fuse.search(searchKeyword);
```

Comment:
how's the perf? maybe we can add memo if needed here

```suggestion
    const results = useMemo(() => fuse.search(searchKeyword), [searchKeyword]);
```